rpc_api.go

```diff
@@ -616,3 +616,9 @@ func (rpcapi *PeerMonitorRPCAPI) LatestMetrics(ctx context.Context, in string, o
 	*out = rpcapi.mon.LatestMetrics(ctx, in)
 	return nil
 }
+
+// MetricNames runs PeerMonitor.MetricNames().
+func (rpcapi *PeerMonitorRPCAPI) MetricNames(ctx context.Context, in struct{}, out *[]string) error {
+	*out = rpcapi.mon.MetricNames()
```

Comment:
```suggestion
	*out = rpcapi.mon.MetricNames(ctx)
```

Reply:
This is in `store`, we dont use context here

Reply:
👍

Reply:
No, you do need to pass the content here